gmeta/codegen/src/lib.rs

```diff
@@ -6,6 +6,8 @@ use syn::{
     Type, TypePath, Visibility,
 };
 
+static MODULE_NAME: &str = "metafns";
```

Comment:
Why `static` instead of `const`?

Reply:
I prefer `static` because it's not copied & pasted like `const` but is saved once in the static memory & used by reference.